app/views/articles/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Nytt Blogginnlegg</h1>
+
+<%= form_for :article, url: articles_path do |f| %>
+	<% if @article.errors.any? %>
+		<div id="error_explanation">
+			<h2>
+				<%= pluralize(@article.errors.count, "grunn") %> til at dette ikke ble lagret:
```

Comment:
`pluralize` doesn't understand Norwegian, unfortunately. If you were using the I18n library here, you could have the following in your Norwegian translation file:

```yml
nb:
  articles:
    new:
      reason:
        one: "grunn"
        other: "grunner"
```

And then used it in your template like this:

```erb
<%= t(".reason", count: @article.errors.count) %>
```

In this particular case though, there are some default translations you could use. Check the [default Norwegian language file](https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/nb.yml#L136-L140).